index.js

```diff
@@ -21,25 +24,25 @@ module.exports = postcss.plugin('postcss-increase-specifity', function(options)
 		css.walkRules(function(rule) {
 			rule.selectors = rule.selectors.map(function(selector) {
 				// Apply it to the selector itself if the selector is a `root` level component
-				// `html:root:root:root`
+				// `html:not(#\\20):not(#\\20):not(#\\20)`
 				if(
 					selector === 'html' ||
-					selector === ':root' ||
+					selector === ':not(#\\20)' ||
```

Comment:
We should put this back to `:root`. `:not(#\\20)` would be covered by the last case `selector === opts.stackableRoot`